syntax = "proto2";

option go_package = "github.com/ydb-platform/nbs/cloud/storage/core/tools/common/go/configurator/kikimr-proto";

message TAuthConfig {
    optional string AccessServiceEndpoint = 1;
    optional string UserAccountServiceEndpoint = 2;
    optional bool UseBlackBox = 3;
    optional bool UseAccessService = 4;
    optional bool UseStaff = 5;
    optional bool UseUserAccountService = 6;
    optional string PathToRootCA = 7;
    optional string AccessServiceType = 8;
}
